server/src/migrations/1745813980616-CreateMissingTagClosures.ts

```diff
@@ -0,0 +1,31 @@
+import { Kysely, sql } from 'kysely';
+import { columns } from 'src/database';
+
+export async function up(db: Kysely<any>): Promise<void> {
+  const tags = await db.selectFrom('tags').select(columns.tag).execute();
+  return db.transaction().execute(async (tx) => {
```

Comment:
We write migrations with raw SQL not the query builder. Please take a look at other migrations. This is more portable in our opinion.